File: ACM Contest Scoring/acm.php
<?php
/**
 * ACM by Martin Björling, 
 * martinbjorling@gmail.com, 0737565044
 *
 * Problem: ACM Contest Scoring
 * https://open.kattis.com/problems/acm
 *
 * @author   Martin Björling
 * @email    martinbjorling@gmail.com
 * @version  1.0
 * @since    2015-11-24
 */

$problemsSolved = 0;
$minutesResult = 0;

$wrongString = "";

while (true) {
	$inputString = stream_get_line(STDIN, 1024, PHP_EOL);

	if ($inputString == "-1") break;

	$inputArr = explode(" ", $inputString);

	$ansTime = $inputArr[0];
	$ansResult = $inputArr[2];

	if ($ansResult == "right") {
		$problemsSolved++;

		$counter = 0;
		for($i = 0; $i < strlen($wrongString); $i++) {
			if ($wrongString[$i] == $inputArr[1]) {
				$counter++;
			} 
		}

		$minutesResult = $minutesResult + $counter*20 + $ansTime;
	} else {
		$wrongString .= $inputArr[1];
	}
}

if ($problemsSolved == 0) $minutesResult = 0;

fwrite(STDOUT, $problemsSolved." ".$minutesResult);

?>

File: ACM Contest Scoring/Acm.java
/**
 * Problem: ACM Contest Scoring
 * https://open.kattis.com/problems/acm
 *
 * @author   Martin Björling
 * @email    martinbjorling@gmail.com
 * @version  1.0
 * @since    2015-11-24
 */

import java.lang.Math;
import java.util.Scanner;
import java.util.ArrayList;

public class Acm {
	
	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		int problemsSolved = 0;
		int minutesResult = 0;

		String wrongString = "";

		while (true) {
			String inputString = sc.nextLine();

			if (inputString.equals("-1")) break;

			String[] inputArr = inputString.split(" ");

			int ansTime = Integer.parseInt(inputArr[0]);
			String ansResult = inputArr[2];

			if (ansResult.equals("right")) {
				problemsSolved++;

				int counter = 0;
				for(int i = 0; i < wrongString.length(); i++) {
				    if (wrongString.charAt(i) == inputArr[1].charAt(0)) {
				        counter++;
				    } 
				}



				minutesResult = minutesResult + counter*20 + ansTime;
			} else {
				wrongString += inputArr[1];
			}
		}
		
		if (problemsSolved == 0) minutesResult = 0;

		System.out.println(Integer.toString(problemsSolved) + " " + Integer.toString(minutesResult));
		
	}
}